### PR TITLE
[HOPSWORKS-CLOUD-138] Upgrade prepartions

### PR DESCRIPTION
**Pull request by maismail**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -261,6 +261,21 @@
     mode 0750
     action :create
   end
+
+   # template all the ddl files from all versions
+   cookbook_file "#{theDomain}/flyway/all/sql/V#{version}__hopsworks.sql" do
+    source "sql/ddl/updates/#{version}.sql"
+    owner node['glassfish']['user']
```
Comment (kouzant) [inline recipes/default.rb:268]: Explicitly set group of file

File: recipes/default.rb [hunk 2]
```diff
@@ -261,6 +261,21 @@
...
+    mode 0750
+    action :create
+  end
+
+  cookbook_file "#{theDomain}/flyway/all/undo/U#{version}__undo.sql" do
+    source "sql/ddl/updates/undo/#{version}__undo.sql"
+    owner node['glassfish']['user']
```
Comment (kouzant) [inline recipes/default.rb:275]: Same here, set group of file

**Review by kouzant — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 1 above)
Comment (maismail) [reply recipes/default.rb:268]: Ok, done. However, I was keeping it the same as other blocks in the recipe https://github.com/logicalclocks/hopsworks-chef/blob/master/recipes/default.rb#L205

Comment (kouzant) [reply recipes/default.rb:268]: I'm not sure if the assigned group would be the primary group of the `owner` or the primary group of the user which runs karamel

**Review by kouzant — APPROVED** [review]

**Pull request merged**